### DISCUSSION TITLE
Project Beta - Display names instead of usernames

### DISCUSSION THREAD

**Discussion by EdwardSalkeld** [community/community#19682] [Projects and Issues]
When using the new projects boards to manage an organisation's work, it'd look a lot better if Profile Name was used instead of Username in the assignee column. People use all sorts of weird and wonderful things in their usenames (or are forced to because their real name is taken), but tend to write their real names in their profile.

![image](https://user-images.githubusercontent.com/107061858/176426870-dc77e202-19b0-4b4f-8f27-465aa4784eac.png)

In case it's not entirely clear (as my username and profile name are similar), I would expect the screenshot to show 'Ed Salkeld' where it currently says 'EdwardSalkeld'

**Comment by p0fi** [comment]
This absolutely an option which should exist. Enable display of Profile Names instead of Username.
